challenges/functions.js

```diff
@@ -6,17 +6,28 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+let  Func = (param1,param2,cb) => {
```

Comment:
Solid use of arrow functions 👍 
However this function's name should be `consume` and not `Func`